packages/js/navigation/src/index.js

```diff
@@ -266,6 +267,42 @@ export const addHistoryListener = ( listener ) => {
 	};
 };
 
+/**
+ * Determines if a URL is a WC admin url.
+ *
+ * @param {*} url - the url to test
+ * @return {boolean} true if the url is a wc-admin URL
+ */
+export const isWCAdmin = ( url = window.location.href ) => {
+	return /admin.php\?page=wc-admin/.test( url );
+};
+
+/**
+ * A utility function that navigates to a page, using a redirect
+ * or the router as appropriate.
+ *
+ * @param {Object} args        - All arguments.
+ * @param {string} [args.url]  - Relative or absolute url to navigate to.
+ * @param {string} [args.path] - Path to navigate to.
+ */
+export const navigateTo = ( { url, path } ) => {
```

Comment:
I find the `path` and `url` params a bit confusing (especially since URLs can be relative).

Is this in here because we might have a fully qualified URL that is actually a WCA page?  Is there any opportunity to combine these into a single argument or do you think that complicates the functionality too much?

Reply:
@joshuatf This is definitely true, and I've ended up revamping the utility to only accept the url parameter. That said, this may require a bit more thought since the way(s) that we're handling these links is a bit all over the place. 

For example, the `actionUrl` for the tasks can be either an absolute URL, or actually just just `path` parameter. The second of which just assumes that it's a WCAdmin page, so we don't have the ability to get a non-wcadmin page via that parameter.

Right now in the `navigateTo` utility I've settled for assuming either an absolute URL or a wc-admin path (starting with `admin.php` as is done with the `getNewPath()` util), with the assumpt that it's relative to `/wp-admin/`. I also left that last bit configurable, but I'm still a bit uncertain how we want to approach all of this. Further work should likely move to another PR, but it's been nice to utilize it for the tasks!